### DISCUSSION TITLE
can i use it with nest js?

### DISCUSSION THREAD

**Discussion by Adamkaram** [urql-graphql/urql#1277] [Q&A] [answered]
i'm newcomer to nest js and i want to use diffrent thing in it like urql does it possible

**Comment by JoviDeCroock** [comment, marked as answer]
`Nest.JS` is a backend framework, while you can use `urql` in a Node environment I don't see much use for it in said environment. The primary use case here is still Frontend frameworks and using it in presentational applications.

**Comment by Adamkaram** [comment]
got it thanks
